main.py

```diff
@@ -1,10 +1,43 @@
+import json
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        players = json.load(file)
+
+        for name, player_info in players.items():
+            race_info = player_info["race"]
+            race, _ = Race.objects.get_or_create(
+                name=race_info["name"],
+                description=race_info["description"]
+            )
+
+            skill_info = race_info["skills"]
+            for skill in skill_info:
```

Comment:
Could you iterate without creating another viriable ?
`for skill in race_info['skills']:`